app/assets/stylesheets/root_path.scss

```diff
@@ -0,0 +1,20 @@
+.parallax-container {
+  background-image: url("https://d2lm6fxwu08ot6.cloudfront.net/img-thumbs/960w/0P9709P1DQ.jpg")  ;
```

Comment:
@AELSchauer @alfosco @JonathanSR i couldn't get this to load from anywhere in the file tree - would love your insight on this!